website/docs/tutorial/getting-set-up/setting-up-databricks.md

```diff
@@ -157,48 +158,90 @@ For dbt to connect to the most optimized SQL experience in Databricks, you need
 
 ## Connecting to dbt Cloud
 
-Now it’s time to connect dbt Cloud to Databricks in order to develop your dbt Project. 
+ There are two ways to connect dbt Cloud and Databricks. The first option is Partner Connect, which provides a streamlined setup to create your dbt Cloud account from within your new Databricks trial account. The second option is to create your dbt Cloud account separately and build the Databricks connection yourself. If you are looking to get started quickly, we recommend option 1. If you are looking to customize your setup from the very beginning and gain familiarity with the dbt Cloud setup flow, we recommend option 2.
 
-<Snippet src="tutorial-create-new-dbt-cloud-account" />
+### Option 1: Connect dbt Cloud and Databricks with partner connect
+
+1. In the Databricks workspace, on the left-side console: click on `Partner Connect`. 
+
+<div style={{maxWidth: '400px'}}>
+<Lightbox src="/img/databricks_tutorial/images/databricks_partner_connect.png" title="Databricks Partner Connect" />
+</div>
 
-### Connect dbt Cloud to Databricks
+2. Select the dbt tile under `Data preparation and transformation`.
+3. Click on `Next` when prompted to `Connect to partner`. This action will create a service principal, PAT token for that service principle, and SQL Endpoint for the dbt Cloud account to use. This does mean that you will have two SQL Endpoints at your disposal from the previous step and from using Partner Connect.
 
-Now let's set up the connection between dbt Cloud and Databricks.
+<div style={{maxWidth: '400px'}}>
+<Lightbox src="/img/databricks_tutorial/images/databricks_connect_to_partner.png" title="Databricks Partner Connect Connect to dbt Cloud" />
+</div>
 
-1. Choose **Databricks** to setup your connection.
+
+4. Click on `Connect to dbt Cloud`.
+<div style={{maxWidth: '400px'}}>
+<Lightbox src="/img/databricks_tutorial/images/databricks_connect_to_dbt_cloud.png" title="Databricks Partner Connect Connect to dbt Cloud" />
+</div>
+5. After the new tab loads, you will be bought to a webform. If you have made a dbt Cloud account with your email previously, you will be asked to provide an account name. If you haven't, you will be asked to provide an account name and password. 
+
+<div style={{maxWidth: '400px'}}>
+<Lightbox src="/img/databricks_tutorial/images/databricks_partner_connect_create_account.png" title="Databricks Partner Connect Connect to dbt Cloud" />
+</div>
+
+6. After you have filled out the form and clicked on `Complete Registration`, you will be logged into dbt Cloud automatically. 
+
+
+### Option 2: Connect dbt Cloud and Databricks manually
+
+1. To get setup in dbt Cloud manually, you will need to get the SQL Endpoint connection information and generate a user token. You can find your SQL endpoint connection information by going to the `Databricks UI > SQL > SQL Endpoints > Starter Endpoint > Connection details`. Save this information or keep this window open for the next section.
```

Comment:
```suggestion
1. To manually setup dbt Cloud, you will need the SQL Endpoint connection information and to generate a user token. You can find your SQL endpoint connection information by going to the `Databricks UI > SQL > SQL Endpoints > Starter Endpoint > Connection details`. Save this information because you will need it later.
```